websites/M/Modrinth/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "424508522542137345",
+		"name": "ダニル"
+	},
+	"service": "Modrinth",
+	"description": {
+		"en": "Search and browse thousands of Minecraft mods on Modrinth with instant, accurate search results!"
+	},
+	"url": [
+		"modrinth.com",
+		"docs.modrinth.com",
+		"blog.modrinth.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/DHtRMas.png",
```

Comment:
Logos must be [at least 512x512px](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).